src/mahoji/commands/finish.ts

```diff
@@ -36,6 +36,32 @@ export const finishCommand: OSBMahojiCommand = {
 			if (val.cl.every(id => loot.has(id))) break;
 			kc++;
 			const thisLoot = val.kill({ accumulatedLoot: loot });
+
+			// This section adds loot that is normally not rolled from the loot table
+			if (val.name === 'Chambers of Xeric (Solo, CM)') {
+				if (kc === 100) thisLoot.add("Xeric's guard", 1);
+				if (kc === 500) thisLoot.add("Xeric's warrior", 1);
+				if (kc === 1000) thisLoot.add("Xeric's sentinel", 1);
+				if (kc === 1500) thisLoot.add("Xeric's general", 1);
+				if (kc === 2000) thisLoot.add("Xeric's champion", 1);
+			}
+			if (val.name.includes('Theatre of Blood')) {
+				if (kc === 100) thisLoot.add('Sinhaza shroud tier 1', 1);
+				if (kc === 500) thisLoot.add('Sinhaza shroud tier 2', 1);
+				if (kc === 1000) thisLoot.add('Sinhaza shroud tier 3', 1);
+				if (kc === 1500) thisLoot.add('Sinhaza shroud tier 4', 1);
+				if (kc === 2000) thisLoot.add('Sinhaza shroud tier 5', 1);
+			}
+			if (val.name === 'Corrupted Gauntlet') thisLoot.add('Gauntlet cape', 1);
+			if (val.name === 'Tempoross') {
+				if (loot.amount('Spirit flakes') >= 4800) {
```

Comment:
Actually going to rework this slightly and pull these out of the loop so it's not being added on every kill after 4800 flakes has been reached.